### DISCUSSION TITLE
[DISCUSSION] In progress status ideas (watch this week's video for info)

### DISCUSSION THREAD

**Discussion by ErikSchierboom** [exercism/v3#1455] [General]
We've discussed some things on figuring out how to make contributing easier and with less friction. Ideas that came up were:

- Draft PR's (wizard)
- Having an incomplete/to-be-improved status field/flag
- Limiting the number of initial files required for a PR to be merged (example + introduction/example + design)

We'd love to hear your thoughts!

**Comment by iHiD** [comment]
We'll take any ideas here and right them up into a proposal on Tuesday. I'll get the video call up over the weekend so people who weren't on it can watch too :)

**Comment by neenjaw** [comment]
Creating a staging process/nomenclature/flag/enum through the config.json so that we can comfortably merge PRs into certain stages to visibly see momentum/progress and better identify concrete tasks for maintainer participation.

Would also help with rapid feedback to be able to see the graph of concept exercises developing to better identify areas for new exercise development

**Comment by iHiD** [comment]
@exercism/track-maintainers Video is here: https://youtu.be/VjKltnkW2qE
